From 06190393dd5991149a117d4630c95e23d390865e Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Wed, 1 Feb 2017 15:01:15 +1000
Subject: [PATCH libinput] Revert "touchpad: reduce the initial timeout for
 tapping after touch"

This reverts commit d0ba1e2b383acaac8a31a1ded7d3039bcd161db3.
---
 src/evdev-mt-touchpad-tap.c | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/src/evdev-mt-touchpad-tap.c b/src/evdev-mt-touchpad-tap.c
index ffd9d0b..03d4c70 100644
--- a/src/evdev-mt-touchpad-tap.c
+++ b/src/evdev-mt-touchpad-tap.c
@@ -33,7 +33,6 @@
 
 #include "evdev-mt-touchpad.h"
 
-#define DEFAULT_TAP_INITIAL_TIMEOUT_PERIOD ms2us(100)
 #define DEFAULT_TAP_TIMEOUT_PERIOD ms2us(180)
 #define DEFAULT_DRAG_TIMEOUT_PERIOD ms2us(300)
 #define DEFAULT_TAP_MOVE_THRESHOLD TP_MM_TO_DPI_NORMALIZED(1.3)
@@ -128,13 +127,6 @@ tp_tap_notify(struct tp_dispatch *tp,
 }
 
 static void
-tp_tap_set_initial_timer(struct tp_dispatch *tp, uint64_t time)
-{
-	libinput_timer_set(&tp->tap.timer,
-			   time + DEFAULT_TAP_INITIAL_TIMEOUT_PERIOD);
-}
-
-static void
 tp_tap_set_timer(struct tp_dispatch *tp, uint64_t time)
 {
 	libinput_timer_set(&tp->tap.timer, time + DEFAULT_TAP_TIMEOUT_PERIOD);
@@ -163,7 +155,7 @@ tp_tap_idle_handle_event(struct tp_dispatch *tp,
 	case TAP_EVENT_TOUCH:
 		tp->tap.state = TAP_STATE_TOUCH;
 		tp->tap.first_press_time = time;
-		tp_tap_set_initial_timer(tp, time);
+		tp_tap_set_timer(tp, time);
 		break;
 	case TAP_EVENT_RELEASE:
 		break;
-- 
2.9.3

